ui/app/src/components/RenameDialogBody/RenameItemView.tsx

```diff
@@ -68,9 +70,20 @@ export function RenameItemView(props: RenameItemViewProps) {
   const { authoringBase } = useEnv();
   const siteId = useActiveSiteId();
   const dispatch = useDispatch();
+  const [legacyFormDialogState, setLegacyFormDialogState] = useSpreadState({
+    open: false,
+    path: null
+  });
 
   const handleEditorDisplay = (item: DetailedItem) => {
-    openItemEditor(item, authoringBase, siteId, dispatch, fetchRenameAssetDependants());
+    if (item.systemType === 'component' || item.systemType === 'page') {
```

Comment:
Why is the handling different based on system type?

Reply:
At this point we may have for example assets/templates/etc, and the else condition handle those that don't use a form

Reply:
Do we have a system type for taxonomy, or does it come as component? Check that the case is handled.

Reply:
There's a system type for taxonomy, updated the validation. Actually there are a couple of details related to taxonomies here, will ping you to see if a new ticket is required for it